app/views/decidim/authorization_modals/_content.html.erb

```diff
@@ -23,14 +23,7 @@
     <% if status.data[:fields] %>
       <ul>
         <% status.data[:fields].each do |field, value| %>
-          <% case field %>
-          <% when :birthdate %>
-            <li> <strong> <%= text_invalid_age_authorizer(status) %> </strong> </li>
-          <% when :subcensus %>
-            <li> <strong> <%= t(".subcensus.invalid_field", scope: "diba")%> </strong> </li>
-          <% else %>
-            <li><strong><%= t ".#{status.code}.invalid_field", field: t("#{status.handler_name}.fields.#{field}", scope: "decidim.authorization_handlers"), value: value ? "(#{value})" : "" %></strong></li>
-          <% end %>
```

Comment:
why are we removing this customization?

Reply:
Mistake, it shouldn't be removed. Reverted changes.